Makefile

```diff
@@ -1,9 +1,13 @@
 VERSION := $(shell git describe --tags --dirty=-modified)
 SERVICE_CONTROLLER_IMAGE := quay.io/skupper/service-controller
 SITE_CONTROLLER_IMAGE := quay.io/skupper/site-controller
+TEST_IMAGE := quay.io/skupper/skupper-tests
 DOCKER := docker
 
-all: build-cmd build-controllers
+all: build-cmd build-controllers build-tests
```

Comment:
Should we include the new target here (as part of the all target)? Once we have dozens
of test apps, we are going to pollute the base directory with test binaries. Maybe we can
put test binaries in a separate place.

Reply:
I agree, the same applies to skupper, skupper-service-controller and skupper-site-controller binaries?

Reply:
skupper binary should be there (imho), as users might be interested in it.

Reply:
Thanks!, I will modify makefile to create a specific folder for containing integration tests binaries

Reply:
Done